test/functional/home_controller_test.rb

```diff
@@ -45,7 +45,7 @@ def setup
     UserSession.create(users(:bob))
     get :dashboard
     assert_response :success
-    assert_select 'a[href=?]', "mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public+Lab&body=Hi,+I+found+this+item+that+looks+like+spam+or+needs+to+be+moderated:+Canon+A1200+IR+conversion+at+PLOTS+Barnraising+at+LUMCON+https://publiclab.org/n/1+by+https://publiclab.org/profile/jeff+Thanks!"
+    assert @response.body.match("mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public+Lab&body=Hi,+I+found+this+item+that+looks+like+spam+or+needs+to+be+moderated:+Canon+A1200+IR+conversion+at+PLOTS+Barnraising+at+LUMCON+https://publiclab.org/n/1+by+https://publiclab.org/profile/jeff+Thanks!")
```

Comment:
OK, i looked this up here: https://guides.rubyonrails.org/testing.html#instance-variables-available and found it needs to be `@response.body`. Let's see!

Reply:
OK, apparently i was wrong somehow! This one is a real tough one! I've tried just comparing it to `''`, which should let us look at the actual text (when it errors) because assert_equal outputs both things you're comparing when there's a problem. Then we can problem-solve. Otherwise we're good to go!